docs/user-guide/deployments/deploy-on-kubernetes/greptimedb-operator-management.md

```diff
@@ -0,0 +1,233 @@
+
+The GreptimeDB Operator includes, but is not limited to, the following features:
+
+- **Automated Provisioning**
+
+  Automates the deployment of the GreptimeDB cluster and standalone on Kubernetes by providing CRD `GreptimeDBCluster` and `GreptimeDBStandalone`.
+
+- **Multi-Cloud Support**
+
+  Users can deploy the GreptimeDB on any Kubernetes cluster, including on-premises and cloud environments(like AWS, GCP, Aliyun, etc.).
+
+- **Scaling**
+
+  Scale the GreptimeDB cluster as easily as changing the `replicas` field in the `GreptimeDBCluster` CR.
+
+- **Monitoring Bootstrap**
+
+  Bootstrap the GreptimeDB monitoring stack for the GreptimeDB cluster by providing the `monitoring` field in the `GreptimeDBCluster` CR.
+
+This document will show you how to install, upgrade, configure, and uninstall the GreptimeDB Operator on Kubernetes.
+
+:::note
+The following output may have minor differences depending on the versions of the Helm charts and environment.
+:::
+
+## Prerequisites
+
+- Kubernetes >= v1.18.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) >= v1.18.0
+- [Helm](https://helm.sh/docs/intro/install/) >= v3.0.0
+
+## Production Deployment
+
+For production deployments, it's recommended to use Helm to install the GreptimeDB Operator.
+
+### Installation
+
+You can refer [Install the GreptimeDB Operator](/user-guide/deployments/deploy-on-kubernetes/getting-started.md#install-the-greptimedb-operator) for detailed instructions.
+
+:::note
+If you are using [Argo CD](https://argo-cd.readthedocs.io/en/stable/) to deploy applications, please make sure that the `Application` has set the [`ServerSideApply=true`](https://argo-cd.readthedocs.io/en/latest/user-guide/sync-options/#server-side-apply) to enable the server-side apply(other GitOps tools may have similar settings).
+:::
+
+### Upgrade
+
+We always publish the latest version of the GreptimeDB Operator as a Helm chart in our official Helm repository.
+
+When the new version of the GreptimeDB Operator is released, you can upgrade the GreptimeDB Operator by running the following commands.
+
+#### Update the Helm repository
+
+```bash
+helm repo update greptime
+```
+
+<details>
+<summary>Expected Output</summary>
+```bash
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "greptime" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+</details>
+
+You can use the following command to search the latest version of the GreptimeDB Operator:
+
+```bash
+helm search repo greptime/greptimedb-operator
+```
+
+<details>
+<summary>Expected Output</summary>
+```bash
+NAME                        	CHART VERSION	APP VERSION  	DESCRIPTION
+greptime/greptimedb-operator	0.2.9        	0.1.3-alpha.1	The greptimedb-operator Helm chart for Kubernetes.
+```
+</details>
+
+#### Upgrade the GreptimeDB Operator
```

Comment:
maybe we can move this title before line 71. When users find out there is a newer version, then can choose to update the Operator.